mkdocs.yml

```
@@ -56,6 +56,7 @@ nav:
       - 'Updating and Rebasing Branches': 'updating-rebasing.md'
       - 'Semantic Commit Messages': 'semantic-commits.md'
       - 'Noise Reduction': 'noise-reduction.md'
+      - 'Automerge configuration and troubleshooting': 'automerge-configuration.md'
```

```suggestion
      - 'Automerging Updates': 'automerge-configuration.md'
```

Try to keep it shorter